org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/JavaLanguageServerPlugin.java

```diff
@@ -126,6 +130,11 @@ public void start(BundleContext bundleContext) throws BackingStoreException {
 
 		// Set the ID to use for preference lookups
 		JavaManipulation.setPreferenceNodeId(PLUGIN_ID);
+		// Initializes default preferences
```

Comment:
that should be part of initializeJDTOptions()

Reply:
initializeJDTOptions() doesn't belong to this class anyway, that should be part of the PreferenceManager

Reply:
I have updated the PR.